src/OmniSharp.Host/Internal/CommandOptionExtensions.cs

```diff
@@ -1,11 +1,11 @@
 using System.ComponentModel;
 using Microsoft.Extensions.CommandLineUtils;
 
-namespace OmniSharp.Host.Internal
+namespace OmniSharp.Internal
 {
-    internal static class CommandOptionExtensions
+    public static class CommandOptionExtensions
     {
-        internal static T GetValueOrDefault<T>(this CommandOption opt, T defaultValue)
+        public static T GetValueOrDefault<T>(this CommandOption opt, T defaultValue)
```

Comment:
That's how the aspnet team does things. ;)  https://github.com/aspnet/Mvc/blob/b3fe473e2971da62208729d792be162a6ba60cc7/src/Microsoft.AspNetCore.Mvc.Formatters.Json/Internal/JsonArrayPool.cs